postgresql/postgresql.rb

```diff
@@ -293,18 +293,13 @@ def monitor_postgresql(pg_servers, group_name)
     pg_servers.each do |mhost|
       return if @interrupted
 
-      thishash = server_db_map[mhost['hostname']]
```

Comment:
do we need to initialize `server_db_map` variable to new hash and put values to it? (check code few lines above)